opba-embedded-starter/src/main/resources/application.yml

```diff
@@ -88,8 +88,8 @@ facade:
       # Redirection URL for consent association
       consent-login:
         page:
-          for-ais: ${facade.urls.embedded-ui-base-url}/ais/{authorizationSessionId}/login?redirectCode={fintechUserTempPassword}
-          for-pis: ${facade.urls.embedded-ui-base-url}/pis/{authorizationSessionId}/login?redirectCode={fintechUserTempPassword}
+          for-ais: ${facade.urls.embedded-ui-base-url}/auth/{authorizationSessionId}/login?redirectCode={fintechUserTempPassword}
```

Comment:
Do not change these please, the key idea is to have explicitly different URLs (even if they load the same component in UI) because for PIS there will be no login form and no registration form in certain use cases that will be enabled only in UI (there should be backend refactoring for that).
PIS and AIS flows are different and open to customization.

Reply:
fixed